### PR TITLE
Remove blank when all parameters are not used and Add ErrJobFuncNotFound

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

I have removed the blank `_` when all the parameters are not used in the target function or method.
And more, I have added the `ErrJobFuncNotFound` to return the `not found error` at the `internal/worker/queue.go`.

### Versions:

- Go Version: 1.19.2
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.8

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

File: internal/errors/worker.go [hunk 1]
```diff
@@ -43,4 +43,9 @@ var (
 	ErrJobFuncIsNil = func() error {
 		return New("JobFunc is nil")
 	}
+
+	// ErrJobFuncIsNil represents a function to generate job function is not found.
+	ErrJobFuncNotFound = func() error {
+		return New("JobFunc is not found")
+	}
```
Comment (vankichi) [inline internal/errors/worker.go:50]: It is used for https://github.com/vdaas/vald/pull/1879/files#diff-2666dcdb36edfd61cbb3e1af4d90cbe4c038636caf249f809fc152a3cfc4d0efR154

File: internal/errors/worker.go [hunk 2]
```diff
@@ -43,4 +43,9 @@ var (
 	ErrJobFuncIsNil = func() error {
 		return New("JobFunc is nil")
 	}
+
+	// ErrJobFuncIsNil represents a function to generate job function is not found.
```
Comment (kevindiu) [inline internal/errors/worker.go:47]:
```suggestion
	// ErrJobFuncNotFound represents a function to generate job function is not found.
```

**Review by kevindiu — APPROVED** [review]

**Review by kpango — DISMISSED** [review]

File: internal/errors/mysql.go [hunk 3]
```diff
@@ -36,63 +34,61 @@ var (
...
+		return Wrapf(ErrMySQLNotFoundIdentity, "error required element not found, uuid: %s", uuid)
 	}
 
-	// NewErrMySQLInvalidArgumentIdentity represents a function to generate an error that the argument is invalid.
-	NewErrMySQLInvalidArgumentIdentity = func() error {
-		return &ErrMySQLInvalidArgumentIdentity{
-			err: New("error mysql invalid argument"),
-		}
+	// NewErrMySQLInvalidArgumentIdentity generates an error that the argument is invalid.
+	NewErrMySQLInvalidArgumentIdentity = &MySQLInvalidArgumentIdentityError{
```
[the remark this answers is not in the document]
Comment (hlts2) [reply internal/errors/mysql.go:41]: @vankichi 
Could you please fix this warning too? 🙏

**Review by hlts2 — APPROVED** [review]
LGTM

**Review by kevindiu — APPROVED** [review]

**Review by kpango — APPROVED** [review]

**Pull request merged**